src/sourmash/cli/sig/intersect.py

```diff
@@ -18,6 +18,8 @@
 in any signatures will be ignored and the output signature will have
 `track_abundance` turned off.
 
+Note: `intersect` only creates one output file, with one signature in it.
```

Comment:
```suggestion
Note: `intersect` only creates one output file, with one consolidated signature from the input files.
```

Reply:
I'm not sure what consolidated here would mean so I'm going to ignore this suggestion ;)